hello.py

```diff
@@ -50,11 +50,11 @@ def autocomplete():
     orflist = []
     for f in filelist:
         syst_name = os.path.basename(f).split('.')[0].upper()
-        std_name = dp.standard_name(syst_name)
+        std_name = np.standard_name(syst_name)
         # TODO: When the reference file is complete, use following line instead
-        # orflist.append( syst_name if dp.is_hypothetical_protein(syst_name) else std_name+"/"+syst_name )
+        # orflist.append( syst_name if np.is_hypothetical_protein(syst_name) else std_name+"/"+syst_name )
         orflist.append( syst_name
-                        if (dp.is_hypothetical_protein(syst_name) or dp.is_unknown_protein(syst_name))
+                        if (np.is_hypothetical_protein(syst_name) or np.is_unknown_protein(syst_name))
```

Comment:
Rather than have lots of these 'when the reference file is complete' todo comments, I'd try to move the alternative logics within the helper methods. So make `is_hypothetical_protein` also return `True` if it's an unknown protein, for instance.

Reply:
This is probably addressed by a further refactoring I did in the last commit - let me know if it's ok with you.